scripts/resume.js

```diff
@@ -0,0 +1,86 @@
+const getResume = filePath => {
+  fetch(filePath)
+    .then(response => {
+      return response.json();
+    })
+    .then(data => {
+      // console.log(data.basics);
+      renderResume(data);
+    });
+};
+
+const renderResume = data => {
+  renderHeader(data.basics);
+  renderContactInfo(data.basics);
+  renderAbout(data.basics);
+  renderProfiles(data.basics.profiles);
+};
+
+///////////////////////////////////////////////////
+//     Resume Section Templates
+///////////////////////////////////////////////////
+// Each function below is a template for a single section
+// of the resume
+
+// The header, containing full name and job title
+const renderHeader = basics => {
+  const html = `
+    <h1>${basics.name}</h1>
+    <h2>${basics.label}</h2>
+  `;
+  $(".header").html(html);
+};
+
+const renderContactInfo = basics => {
+  const html = `
+    <aside>Contact</aside>
+    <div>
+      <p><strong>Email</strong>: ${basics.email}</p>
+      <p><strong>Phone:</strong> ${basics.phone}</p>
+      <p><strong>Website:</strong> <a href="${basics.website}">${basics.website}</a>
+      </p>
```

Comment:
Would problem create a line break for your <a> tag so that it creates a bit of hierarchy in your code with the <p> tags.